config/locales/en.yml

```diff
@@ -93,18 +93,18 @@ en:
           <a href='%{privacy_url}'>Privacy Policy</a>
       is_private: "Check to keep your profile private."
       account_information: "Account Information"
-      display_name: "Username:"
+      display_name: "Membername:"
```

Comment:
I thought about it for a second, and I am indifferent. On one hand I think it is more consistent with the change to 'members'. On the other, it sounds weird because it is not common, though I don't mind being different and weird :)

Do you have a preference?